moncli/types.py

```diff
@@ -282,6 +281,56 @@ def validate_hour(self,value):
             raise ValidationError('Minute values must be between 0-59, not "{}".'.format(value.minute))
 
 
+class ItemLinkType(MondayType):
+
+    native_type = list
+    native_default = []
+    null_value = {}
+    multiple_values = True
+    allow_casts = (int,str)
+
+    def __init__(self, id: str = None, title: str = None, multiple_values: bool = True, *args, **kwargs):
+        self.multiple_values = multiple_values
+        if not multiple_values:
+            self.native_type = int
+            self.native_default = None
+        super().__init__(id, title, *args, **kwargs)
+
+    def _process(self, value):
+        if not self.multiple_values:
+            try:
+                return_value =  int(value) if isinstance(value,(int,str)) else int(value[0])
+                return return_value
+            except ValueError:
+                raise ConversionError('Invalid item ID "{}".'.format(value))
+        elif self.multiple_values:
+            return_list = []
+            if not isinstance(value,list):
+                return_list = int(value)     
+            for data in value:
+                try:
+                    return_list.append(int(data))
+                except ValueError:
+                    raise ConversionError('Invalid item ID "{}".'.format(data))
+            return return_list
+            
+    def _process_column_value(self, value: en.cv.ColumnValue):
+        try:
+            if value.settings['allowMultipleItems']:
+                self.multiple_values = value.settings['allowMultipleItems']
+            if not self.multiple_values:
+                return value.value[0] if value.value else self.native_default
```

Comment:
Before returning the value, we will need to do down here what we did at the top.  

This means that here, we will also need to set _element_type_ to None and _allow_casts_ to **(str,)**. 

And while you are here, it would also be best to set the native_type to int and _native_default_ to None here as well, just in case the _multiple_values_ setting was missed when creating the type field.